service/collector/issue_test.go

```diff
@@ -0,0 +1,112 @@
+package collector
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/giantswarm/to"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-github/github"
+)
+
+func Test_Collector_Issue_hasLabels(t *testing.T) {
```

Comment:
`hasLabels` is a standalone function and has to be unique in scope of the package.

```suggestion
func Test_hasLabels(t *testing.T) {
```

Reply:
The structured test name is on purpose so you have an idea what it is about and where it comes from. I do not think the suggestion here is better.